### ISSUE TITLE
Using println breaks when cargo-expand uses rustfmt

### ISSUE THREAD

**Issue by JelteF** [dtolnay/cargo-expand#6]
For finding out where I was generating wrongly formatted quotes I used `println('{:?}', tokens)` to find out what tokens were being generated. Leaving those prints in causes `cargo expand` to hiccup when rustfmt is in the path, because they are seen as the first part of the code and it's not valid Rust.

```
error: expected item, found `Tokens`
 --> stdin:1:1
  |
1 | Tokens("MyInt ( self . 0 . add ( rhs . 0 ) )")
  | ^^^^^^

```

This can be worked around by using `RUSTFMT= cargo expand`, to disable rustfmt.

**Comment by dtolnay** [comment]
Thanks for reporting this! I fixed this in cargo-expand 0.3.6.

**Comment by JelteF** [comment]
Great!
